build.gradle

```diff
@@ -4,24 +4,29 @@ plugins {
     id 'maven'
 }
 
-sourceCompatibility = '1.7'
+group = 'uk.sky.cirrus'
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenCentral()
+}
 
 task wrapper(type: Wrapper) {
     gradleVersion = '2.7'
 }
 
 dependencies {
-    compile 'com.datastax.cassandra:cassandra-driver-core:2.0.3'
+    compile 'com.datastax.cassandra:cassandra-driver-core:2.1.8'
```

Comment:
What will happen if someone else depends on a different driver version?

I wonder if we should consider encapsulating the driver (via jarjar/proguard) if it's not too large. But that has its own complexities.